helm-rhythmbox.el

```diff
@@ -120,6 +120,15 @@ formatted with `helm-rhythmbox-candidate-format'."
       (dbus-call-method :session service path interface
                         "AddToQueue" (rhythmbox-song-uri song)))))
 
+(defun helm-rhythmbox-pause-song ()
```

Comment:
I tend to read slashes as module separators, so ```helm-rhythmbox-play/pause-song``` reads funny to me.  Maybe just ```helm-rhythmbox-playpause-song```?